src/styles/styleGenerators.js

```diff
@@ -0,0 +1,88 @@
+// Create spacing styles genorator for given property.
```

Comment:
This file is where the functions that generate styles live. Will probably break up as size increases.